<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometrySettingsDialog</class>
 <widget class="QDialog" name="GeometrySettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="mCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mOkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="mOptionsListWidget">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>CG Movement Options</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aerodynamics Options</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="mStackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mCGOptionsPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="mMoveCGAllButton">
         <property name="text">
          <string>Move To CG (All axes) [F4]</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="mMoveCGXZButton">
         <property name="text">
          <string>Move To CG (X and Z axes) [F3]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mMoveCGZButton">
         <property name="text">
          <string>MoveTo CG (Z-axis only) [F2]</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="mMoveCGXButton">
         <property name="text">
          <string>Move To CG (X-axis only) [F1]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mAeroOptionsPage">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="7" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="9" column="1">
          <widget class="QLineEdit" name="mMinAlphaLineEdit"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="mMinAlphaLabel">
           <property name="text">
            <string>Min Alpha</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QLineEdit" name="mMaxAlphaLineEdit"/>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="mMaxAlphaLabel">
           <property name="text">
            <string>Max Alpha</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="mTestAltitudeLineEdit"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="mTestAltitudeLabel">
           <property name="text">
            <string>Test Altitude</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreaseControlAngleButton">
           <property name="text">
            <string>Decrease Control Angle [,]</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="mMaxBetaLabel">
           <property name="text">
            <string>Max Beta</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QLineEdit" name="mMaxBetaLineEdit"/>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="mMaxMachLabel">
           <property name="text">
            <string>Max Mach</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="mTestMachLabel">
           <property name="text">
            <string>Test Mach</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreaseControlAngleButton">
           <property name="text">
            <string>Increase Control Angle [.]</string>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QLineEdit" name="mMaxMachLineEdit"/>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="mTestMachLineEdit"/>
         </item>
         <item row="5" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreaseRollButton">
           <property name="text">
            <string>Decrease Roll Rate [PgDown]</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreaseSpeedButton">
           <property name="text">
            <string>Decrease Speed [-]</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreasePitchButton">
           <property name="text">
            <string>Decrease Pitch Rate [End]</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreaseYawButton">
           <property name="text">
            <string>Increase Yaw Rate [Ins]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreaseBetaButton">
           <property name="text">
            <string>Increase Vehicle Beta [3]</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreaseYawButton">
           <property name="text">
            <string>Decrease Yaw Rate [Del]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreaseBetaButton">
           <property name="text">
            <string>Decrease Vehicle Beta [1]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QPushButton" name="mDecreaseAlphaButton">
           <property name="text">
            <string>Decrease Vehicle Alpha [8]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreaseAlphaButton">
           <property name="text">
            <string>Increase Vehicle Alpha [2]</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreaseRollButton">
           <property name="text">
            <string>Increase Roll Rate [PgUp]</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreasePitchButton">
           <property name="text">
            <string>Increase Pitch Rate [Home]</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QPushButton" name="mIncreaseSpeedButton">
           <property name="text">
            <string>Increase Speed [+]</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QCheckBox" name="mAlphaSymmetryCheckBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="mAlphaSymmetryLabel">
           <property name="text">
            <string>Alpha Symmetry</string>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QCheckBox" name="mBetaSymmetryCheckBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLabel" name="mBetaSymmetryLabel">
           <property name="text">
            <string>Beta Symmetry</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mOptionsListWidget</tabstop>
  <tabstop>mMoveCGXButton</tabstop>
  <tabstop>mMoveCGZButton</tabstop>
  <tabstop>mMoveCGXZButton</tabstop>
  <tabstop>mMoveCGAllButton</tabstop>
  <tabstop>mOkButton</tabstop>
  <tabstop>mCancelButton</tabstop>
  <tabstop>mIncreaseAlphaButton</tabstop>
  <tabstop>mDecreaseAlphaButton</tabstop>
  <tabstop>mIncreaseBetaButton</tabstop>
  <tabstop>mDecreaseBetaButton</tabstop>
  <tabstop>mIncreaseYawButton</tabstop>
  <tabstop>mDecreaseYawButton</tabstop>
  <tabstop>mIncreasePitchButton</tabstop>
  <tabstop>mDecreasePitchButton</tabstop>
  <tabstop>mIncreaseRollButton</tabstop>
  <tabstop>mDecreaseRollButton</tabstop>
  <tabstop>mIncreaseSpeedButton</tabstop>
  <tabstop>mDecreaseSpeedButton</tabstop>
  <tabstop>mIncreaseControlAngleButton</tabstop>
  <tabstop>mDecreaseControlAngleButton</tabstop>
  <tabstop>mTestAltitudeLineEdit</tabstop>
  <tabstop>mMaxBetaLineEdit</tabstop>
  <tabstop>mMinAlphaLineEdit</tabstop>
  <tabstop>mMaxAlphaLineEdit</tabstop>
  <tabstop>mTestMachLineEdit</tabstop>
  <tabstop>mMaxMachLineEdit</tabstop>
  <tabstop>mAlphaSymmetryCheckBox</tabstop>
  <tabstop>mBetaSymmetryCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
